### ISSUE TITLE
Make training recommender asyncronous

### ISSUE THREAD

**Issue by Rentier** [inception-project/inception#525]
**Is your feature request related to a problem? Please describe.**
Right now,  the training of recommenders in INCEpTION is syncronous, i.e. recommenders are trained one after another. This is bad, because recommender that need longer to train can slow the whole training process.

**Describe the solution you'd like**
Make the recommender training asynchronous, give the training process an indication when the training is done.

**Comment by reckart** [comment]
@Rentier hasn't this already been addressed by the scheduler refactoring?

**Comment by Rentier** [comment]
More or less. I also wanted running recommenders for a user in parallel, but that is far away.

**Comment by reckart** [comment]
So we have two options. Changing the description of this issue for the far away goal or resolving it as a duplicate of the issue under which the schedule refactoring was addressed and opening a new issue for the far away goal.

**Comment by Rentier** [comment]
I edited this issue.
